battleship.py

```diff
@@ -199,6 +378,14 @@ def runGameTurn(data, row, col):
 Returns: list of ints
 '''
 def getComputerGuess(board):
+    i=0
+    while(i<1):
+        row=randint(0,9)
+        col=randint(0,9)
+        if(board[row][col]==SHIP_UNCLICKED or board[row][col]==EMPTY_UNCLICKED):
+            i=i+1
+            return [row,col]
+
     return

```

Comment:
Instead of using a separate variable for while loop, write the condition of while based on board (like you have written for if and remove if)